javascript/channel.ts

```diff
@@ -0,0 +1,181 @@
+
+import * as util from "./util.ts";
```

Comment:
VSCode tells me:
```
An import path cannot end with a '.ts' extension. Consider importing './util' instead.
```
Should these be changed, or is there a setting I should update for this import convention?